packages/integration_test/lib/integration_test_driver_extended.dart

```diff
@@ -14,13 +14,7 @@ import 'package:path/path.dart' as path;
 
 import 'common.dart';
 
-/// Flutter Driver test output directory.
-///
-/// Tests should write any output files to this directory. Defaults to the path
-/// set in the FLUTTER_TEST_OUTPUTS_DIR environment variable, or `build` if
-/// unset.
-String testOutputsDirectory =
-    Platform.environment['FLUTTER_TEST_OUTPUTS_DIR'] ?? 'build';
```

Comment:
I noticed that the duplicate code was in 2 places: [`integration_test_driver.dart`](https://github.com/flutter/flutter/blob/27d01bee4558ba71f0407384f5e8e320ee8b7cb0/packages/integration_test/lib/integration_test_driver.dart#L17-L23) and [`integration_test_driver_extended.dart`](https://github.com/flutter/flutter/blob/27d01bee4558ba71f0407384f5e8e320ee8b7cb0/packages/integration_test/lib/integration_test_driver_extended.dart#L17). Maybe it'd be ideal to apply this change in both spots.

Reply:
Good catch, I can do the same for that one